src/main/java/tr/com/srdc/cda2fhir/transform/ResourceTransformerImpl.java

```diff
@@ -1737,19 +1732,19 @@ public EntryResult tImmunizationActivity2Immunization(ImmunizationActivity cdaIm
 				}
 			}
 		}
-
-		// negationInd -> notGiven
-		if (cdaImmunizationActivity.getNegationInd() != null) {
-			fhirImmunization.setNotGiven(cdaImmunizationActivity.getNegationInd());
-		}
+////In FHIR R4, there is no concept of 'notGiven', commenting this out in the meantime
```

Comment:
Would this not correspond with the status code of "not-done"?

Reply:
yes. You are correct. im making the changes now